lib/evmone/instructions_traits.hpp

```diff
@@ -177,6 +177,12 @@ struct Traits
     /// The instruction name;
     const char* name = nullptr;
 
+    /// Size of the immediate argument in bytes.
+    uint8_t immediate_size = 0;
+
+    /// Whether the instruction terminates execution.
+    bool is_terminating = false;
```

Comment:
This is not currently the case, but all undefined instructions may be considered "terminating".

Reply:
It does not matter for unimplemented in EOF context, because unimplemented are handled separately. Not used in other places.